### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by jose1711**

### CODE REVIEW

File: src/gui/res/lang/gui_sk-SK.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="sk_SK" sourcelanguage="en">
 <context>
     <name>AboutDialogBase</name>
     <message>
         <location filename="res/AboutDialogBase.ui" line="38"/>
         <source>About Synergy</source>
-        <translation type="unfinished"></translation>
+        <translation>O Synergy</translation>
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/gui/res/lang/gui_sk-SK.ts]: That's a letter 'O'.

File: src/gui/res/lang/gui_sk-SK.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?><!DOCTYPE TS><TS language="sk-SK" sourcelanguage="en" version="2.0">
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="sk_SK" sourcelanguage="en">
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/gui/res/lang/gui_sk-SK.ts]: Linguist changed it automatically. But AFAIK `sk_SK` is somewhat standard for other programs (maybe that is the reason why it got changed). Also this is output of `locale` on my system:
```
LANG=sk_SK.UTF-8
LC_CTYPE="sk_SK.UTF-8"
LC_NUMERIC="sk_SK.UTF-8"
LC_TIME="sk_SK.UTF-8"
LC_COLLATE="sk_SK.UTF-8"
LC_MONETARY="sk_SK.UTF-8"
LC_MESSAGES="sk_SK.UTF-8"
LC_PAPER="sk_SK.UTF-8"
LC_NAME="sk_SK.UTF-8"
LC_ADDRESS="sk_SK.UTF-8"
LC_TELEPHONE="sk_SK.UTF-8"
LC_MEASUREMENT="sk_SK.UTF-8"
LC_IDENTIFICATION="sk_SK.UTF-8"
```

File: src/gui/res/lang/gui_sk-SK.ts [hunk 3]
```diff
@@ -26,701 +28,720 @@ Synergy is based on CosmoSynergy by Richard Lee and Adam Feder.&lt;br /&gt;
...
     <message>
         <location filename="src/MainWindow.cpp" line="435"/>
         <source>Do you trust this fingerprint?
 
 %1
 
-This is a server fingerprint. You should compare this fingerprint to the one on your server's screen. If the two don't match exactly, then it's probably not the server you're expecting (it could be a malicious user).
+This is a server fingerprint. You should compare this fingerprint to the one on your server&apos;s screen. If the two don&apos;t match exactly, then it&apos;s probably not the server you&apos;re expecting (it could be a malicious user).
 
 To automatically trust this fingerprint for future connections, click Yes. To reject this fingerprint and disconnect from the server, click No.</source>
```
[the remark this answers is not in the document]
Comment (jose1711) [reply src/gui/res/lang/gui_sk-SK.ts]: Again `linguist` did it.

File: src/gui/res/lang/gui_sk-SK.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jnewbon) [reply src/gui/res/lang/gui_sk-SK.ts]: The language selector changes fine so no reason not to change it, only problem is not it doesn't match other language files. Ill accept it anyway as it doesnt affect usage.

File: src/gui/res/lang/gui_sk-SK.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jnewbon) [reply src/gui/res/lang/gui_sk-SK.ts]: Ill have a look in the morning with a built app see if this affects the translation at all. If not ill get this merged in tomorrow.

**Comment by Jnewbon** [comment]
As I don't know the language your translating I cant review the translations themselves so I'm accepting them as is.

File: src/gui/res/lang/gui_sk-SK.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jnewbon) [reply src/gui/res/lang/gui_sk-SK.ts]: App builds fine with them.

**Comment by nbolton** [comment]
Fixes #6622

**Pull request merged**